fluent-bit-to-loki/pkg/config/config.go

```diff
@@ -199,6 +249,7 @@ func ParseConfig(cfg Getter) (*Config, error) {
 			return nil, fmt.Errorf("failed to parse Timeout: %s", timeout)
 		}
 		res.ClientConfig.Timeout = time.Duration(t) * time.Second
+		fmt.Println(res.ClientConfig.Timeout.String())
```

Comment:
Is this necessary or it was a debut print?

Reply:
it was for debug